vNext/src/Orleans/Orleans.csproj

```diff
@@ -33,6 +33,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="1.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0-preview2-final" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0-preview2-final" />
```

Comment:
It works. But be aware that only people with latest VS/Nuget will be able to add `netstandard2.0` libraries on .Net Full framework projects.